### PR TITLE
Update Kleopatra docs for Tails 5.1

### PR DESCRIPTION
**Pull request by eloquence**
## Status

Ready for review

## Description of Changes

Tails 5.1 includes a bugfix for this issue: https://gitlab.tails.boum.org/tails/tails/-/issues/18967

Therefore, we can reasonably remove the workaround instructions that are specific to Tails 5.0, but seems appropriate to still give folks a heads-up that the UX is a bit different.

## Checklist (Optional)

- [x] Doc linting (`make docs-lint`) passed locally
- [ ] Doc link linting (`make docs-linkcheck`) passed
- [x] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

File: docs/upgrade/2.3.2_to_2.4.0.rst [hunk 1]
```diff
@@ -79,28 +79,29 @@ until a fix is available (expected on May 31, 2022). If you are using Tor Browse
 for non-SecureDrop browsing, we recommend restarting Tor Browser before and after using it
 for SecureDrop.
 
-Upgrade to Tails 5.0
---------------------
+Upgrade from Tails 4 to Tails 5 series
```
Comment (zenmonkeykstop) [inline docs/upgrade/2.3.2_to_2.4.0.rst:82]: nit: can probably just say `Tails 5`

Comment (eloquence) [reply docs/upgrade/2.3.2_to_2.4.0.rst]: shortened in headline, kept verbose in body

**Review by zenmonkeykstop — CHANGES_REQUESTED** [review]
One nit, but otherwise LGTM!

File: docs/upgrade/2.3.2_to_2.4.0.rst (hunk 1 above)
Comment (zenmonkeykstop) [reply docs/upgrade/2.3.2_to_2.4.0.rst]: works for me.

**Review by zenmonkeykstop — APPROVED** [review]

**Pull request merged**